core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -657,7 +667,13 @@ var ExplorationEditorMainTab = function() {
     expect(interactionTab(INTERACTION_ID_TO_TAB_NAME[interactionId])
       .isDisplayed()).toBe(true);
     interactionTab(INTERACTION_ID_TO_TAB_NAME[interactionId]).click();
-    expect(interactionTile(interactionId).isDisplayed()).toBe(true);
+
+    var targetTile = interactionTile(interactionId);
+    waitFor.elementToBeClickable(
+      targetTile,
+      'Interaction tile ' + interactionId + ' takes too long to be clickable'
+    );
+    expect(targetTile.isDisplayed()).toBe(true);
```

Comment:
Should we firstly check it's visible then clickable?

Reply:
Update